Framework.Core/System/Reflection/ParameterInfoTheraotExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿#if NETFRAMEWORK && !NET45_OR_GREATER
+namespace System.Reflection;
+
+public static class ParameterInfoTheraotExtensions
+{
+    // TODO: DefaultValue may throw FormatException or return null for structs or Type.Missing
```

Comment:
Not sure what we do with this TODO :)
Do we need it ? Why can't it be simply documented ?

Reply:
I left it as form of documentation. We can handle `FormatException` here, but I think we should try to have a bug to bug conformance. i.e. upgrading tfm 4.0 -> 4.5 should not introduce user to old-new bugs from runtime.
On the other side minor upgrades are not expected within our user group (more likely user will switch to core from netfx, than upgrade netfx tfm)

Reply:
Switching from .NET Framework to .NET is not an easy task. So I would say the .NET Framework upgrade is more probable to happen.

Regarding the docs. Just confer it to 
/// <remarks>

Leaving TODO in code creates too much noise afterwards when you analyzing all TODOs.